CMakeLists.txt

```diff
@@ -372,20 +365,24 @@ if (HCC_INTEGRATE_ROCDL)
   set(ROCDL_BUILD_DIR "${PROJECT_BINARY_DIR}/rocdl")
   file(MAKE_DIRECTORY ${ROCDL_BUILD_DIR})
 
-  # custom commands to build rocdl
-  add_custom_target(rocdl
-    COMMAND ${CMAKE_COMMAND} ${ROCDL_SRC_DIR}
-            -DLLVM_DIR=${CLANG_BIN_DIR}
-            -DAMDHSACOD=${AMDHSACOD}
-            -DGENERIC_IS_ZERO=ON
-            -DBUILD_HC_LIB=ON
-    COMMAND make -j ${NUM_BUILD_THREADS} # not portable, but we need it this way
-    WORKING_DIRECTORY ${ROCDL_BUILD_DIR}
-    DEPENDS clang_links llvm-link
-  )
+  set(LLVM_DIR ${CLANG_BIN_DIR})
+  set(AMDHSACOD ${AMDHSACOD})
+  set(GENERIC_IS_ZERO ON)
```

Comment:
Turns out `set(GENERIC_IS_ZERO ON CACHE BOOL ON FORCE)` is the proper way to ensure the flag is passed down to `ROCDL`.

Also this PR breaks `change-addr-space.sh` within `ROCDL`. I've posted another commit to fix it temporarily and would submit for formal review in `ROCDL`:
https://github.com/whchung/ROCm-Device-Libs/commit/96375f63f45383ced9328221e6cbaccf0e06b13d